src/Kestrel.Transport.Abstractions/Internal/ListenType.cs

```diff
@@ -9,6 +9,8 @@ namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions.Internal
     public enum ListenType
     {
         IPEndPoint,
+        Localhost,
```

Comment:
This makes me super sad

Reply:
I understand the feeling. Still better than "Prefix" though.

Reply:
I thought about it, I think this is too big of a hack to exist. It doesn't fit the current model. This enum needs to be something else and not part of ListenType.

Reply:
Such as? I could drive from Listen options and type check. Or add a second enum IPMode with values LocalhostSplit, AnyIPFallback, and Single. Or derive from ListenOptions and put the bind method on that type. Hmm, that might work...